src/index.js

```diff
@@ -6,6 +6,8 @@ const { promisify } = require('util');
 const { prepareChangeLog } = require('./change_log');
 const { createOrUpdatePullRequest } = require('./pr_helper');
 
+const CHANGELOG_FILE_DESTINATION = 'changelog.txt';
```

Comment:
Maybe this could be nice
```suggestion
const CHANGELOG_FILE_DESTINATION = core.getInput('changelog-file-destination') || 'changelog.txt';
```